packages/orchestration/src/utils/mockChainInfo.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file Mocked Chain Info object until #8879
+ */
+// @ts-check
+import {
+  Order,
+  State as IBCChannelState,
+} from '@agoric/cosmic-proto/ibc/core/channel/v1/channel.js';
+import { State as IBCConnectionState } from '@agoric/cosmic-proto/ibc/core/connection/v1/connection.js';
+
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {CosmosChainInfo, IBCConnectionInfo} from '../cosmos-api.js';
+ */
+
+/**
+ * currently keyed by ChainId, as this is what we have
+ * available in ChainAddress to determine the correct IBCChannelID's
```

Comment:
:+1: 

I'm more comfortable relying on chainid since learning that they're available in-protocol (client state).

I don't think the protocol guarantees uniqueness across the interchain, so when we query for client state, we'll have to be careful to ... um... do something if we find a collision.

Reply:
Ok cool. The only potential hiccup I see, if human readability of the identifier is a design constraint, is that evm chain ids are typically integers.